include/crm/cluster.h

```diff
@@ -201,14 +207,15 @@ static inline const char *
 crm_join_phase_str(enum crm_join_phase phase)
 {
     switch (phase) {
+        case crm_join_nack_quiet:   return "nack_quiet";
```

Comment:
You probably want this to be "nack" -- it's added to the CIB node_state as join=

Reply:
Where? All of these are only for log messages, best I can see:

```
daemons/controld/controld_join_dc.c:51:                  node->uname, current_join_id, crm_join_phase_str(last),
daemons/controld/controld_join_dc.c:58:                 node->uname, current_join_id, crm_join_phase_str(phase),
daemons/controld/controld_join_dc.c:59:                 crm_join_phase_str(last), node->id, source);
daemons/controld/controld_join_dc.c:64:                 current_join_id, node->uname, crm_join_phase_str(last),
daemons/controld/controld_join_dc.c:65:                 crm_join_phase_str(phase), node->id, source);
daemons/controld/controld_join_dc.c:158:                 crm_join_phase_str(member->join));
daemons/controld/controld_join_dc.c:601:                 join_id, join_from, crm_join_phase_str(peer->join),
daemons/controld/controld_join_dc.c:602:                 crm_join_phase_str(crm_join_finalized));
daemons/controld/controld_join_dc.c:667:                      crm_join_phase_str(join_node->join), current_join_id);
daemons/controld/controld_join_dc.c:824:                   crm_join_phase_str(peer->join));
include/crm/cluster.h:207:crm_join_phase_str(enum crm_join_phase phase)
```

Reply:
Ah, I'm wrong -- node_state join can only be down or member (create_node_state_update())